astropy/table/table.py

```diff
@@ -958,9 +975,18 @@ def show_in_browser(self, max_lines=5000, jsviewer=False,
 
         with open(path, 'w') as tmp:
             if jsviewer:
-                self.write(tmp, format='jsviewer', css=css,
-                           max_lines=max_lines, jskwargs=jskwargs,
-                           table_id=tableid, table_class=table_class)
+                has_row_index = 'row_index' in self.colnames
```

Comment:
On frist glance it looks like this is a repeat of previous code and could be factored out.